q2_feature_table/_summarize/summarize_assets/overview.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="./q2templateassets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./q2templateassets/css/normalize.css">
+    <script src="./app/js/child.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <div class="row">
```

Comment:
So currently there is a tiny scrollbar on the bottom of each tab, it seems to be caused by the negative margin of `row`. I think we can remove the class (or set `margin: 0px`) for each row-div on each subpage. Just dropping the class didn't seem to have any ill-effects and did fix the bottom scrollbar. I think this is just an artifact of having an iframe in an iframe with bootstrap.